app/admin/page.tsx

```diff
@@ -37,6 +38,36 @@ function getUsers(props: {pageIndex:number}): Promise<User[]> {
     })
 }
 
+function deleteSandbox(row: Row<any>) {
+  let user:User = row.original
+
+  if (!user.task_arn){
+    toast({
+      title: "Error",
+      description: "No task_arn found",
+    })
+    return
+  }
+
+  fetch(`/api/db/${encodeURIComponent(user.task_arn)}`, {
+    method: 'DELETE',
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  })
+    .then((result) => {
+      if (result.status < 300) {
+        return result.json()
+      }
+      toast({
+        title: "Error",
+        description: result.text(),
```

Comment:
not sure the user will understand what to do with this error

Reply:
this is the error returned from the server